usecase-2.1/README.md

```diff
@@ -133,3 +132,30 @@ scramble inputdir outputdir stub
 scramble inputdir outputdir tsv permute -s participants.tsv
 scramble inputdir outputdir json -p '.*' -s participants.json
 ```
+
+# DatLeak
+
+While working with scrambled data, you can ensure that to what degree intended patterns or information are leaked from the original dataset where you can use [DatLeak](https://github.com/SIESTA-eu/DatLeak) repository. This repository provides a method to test for potential data leakage, checking whether the scrambled variables still contain any identifiable patterns that could be traced back to the original participants. Running DatLeak on scrambled datasets helps confirm that the anonymization process is robust and protects participant privacy.
+
+DatLeak detects data leakage in anonymized datasets by comparing the original data with the scrambled version. It calculates percentage of full leakage (where all variables in a row match) and partial leakage (where some, but not all, variables match). These calculation help assess the effectiveness of the anonymization process.
+
+Assuming you are in **usecase-2.1** directory. Clone the repo:
+```console
+cd ../../
+git clone https://github.com/SIESTA-eu/DatLeak.git && cd DatLeak/
+```
+
+### Usage
+
+```console
+python DatLeak.py test_files/data_original.tsv test_files/data_scramble.tsv -999 
+```
+
+### Ouput 
+
+- Partial Leakage: The percentage of rows with partial leakage.
```

Comment:
can you show the actual output and can you explain how that output is to be interpreted? Someone (i.e., the data rights holder or the platform owner) needs to make a decision whether the data is properly scrambled, and that person needs guidance.

Reply:
I have executed it on the actual data and included the following in the README

```
Partial Leakage: 18.33%
Full Leakage: 81.67%
Average Matching Cells per Row: 254.03
Standard Deviation of Matching Cells per Row: 10.15
```